subjects/java/checkpoints/breakdown-url/README.md

```diff
@@ -0,0 +1,52 @@
+## URL Parsing
+
+### Instructions
+
+Create a class `UrlParser` that provides a method to parse and validate URLs using regex. The method should extract and return url components the `protocol`, `domain`, `port`, `path` and `query` parameters. The URL is always correct.
```

Comment:
url should be all in capital letters to keep the text consistent

Reply:
done